### ISSUE TITLE
Integration With Existing AWS CLI IAM User

### ISSUE THREAD

**Issue by elliottower** [aws-amplify/amplify-cli#4586]
**Is your feature request related to a problem? Please describe.**
In order to sign into the Amplify CLI, all the documentation says you need to create an IAM account or give the info there, but for my case we use Azure SSO for logging in to the regular AWS CLI using [aws azure login](https://github.com/sportradar/aws-azure-login). This creates temporary 1 hour logins which are saved to the credentials file. In order to get through the Amplify configuration, I had to manually copy over this temporary data the first time, and then when it asks which account to use I chose default. Now I don't have to re-copy over the data, but simply run the aws-azure-login command to reset the local copy. So clearly Amplify is able to read the default AWS account and use it, but for the onboarding installation for the CLI it does not allow this. There should be an easier way to install the Amplify CLI, and the installer should be able to simply look for the already existing AWS accounts with the AWS CLI. 

**Describe the solution you'd like**
Integrate the AWS accounts between the AWS CLI and AWS Amplify CLI or simply fix the installer so that it doesn't automatically ask for the secret access key and such to be copied over into the prompt, but instead to look for existing accounts with AWS. In the further commands that I ran, over an hour later once the temporary token that I originally gave the info for expired, I was able to simply choose the default user instead of the new user that I created (and subsequently closed the tab for and cancelled because my company forbids creation of new IAM users.

**Describe alternatives you've considered**
The aforementioned 'hack' of simply copying over the two secret codes from the credentials file which was created from AWS CLI works, but is inconvenient and not a good solution long term

**Additional context**
Add any other context, an idea of the implementation or screenshots about the feature request here.

**Comment by attilah** [comment]
@elliottower the Amplify CLI has no dependency on the AWS CLI, so we cannot use that as a check for credentials as you suggested, but I see the problem you're facing with. We are planning to rework our init workflow and this enhancement will be tracked as part of that work. Thanks for sharing this feedback.

**Comment by elliottower** [comment]
That's weird because I am fairly sure that the credentials I entered in to the amplify CLI were definitely not permanently valid ones, I just copied the 1 hour temporary credentials in and then once the amplify CLI tried to use them after an hour and they didn't work it asked which aws account I want to use: the one I created (through the initialization configure process, except cancelled so it isn't an actual AWS iam user) or default. Then it has continued to work as I've refreshed and ran aws-azure-login despite having different temporary credentials every time. I would expect it would need me to copy them in manually every time but it seems to be able to figure out what accounts are already registered or active in the aws credentials file.

Basically even though it may not be built on the regular AWS CLI I'm pretty sure the integration works currently so that it can read what user account you currently have saved in your credentials file, so I feel like that should be an option when doing the first onboarding setup phase for amplify CLI.
